src/Resources/config/config.yml

```diff
@@ -1,43 +1,27 @@
+imports:
+    - "@LoevgaardSyliusBrandPlugin/Resources/config/grids/loevgaard_sylius_brand_admin_brand.yml"
+    - "@LoevgaardSyliusBrandPlugin/Resources/config/grids/loevgaard_sylius_brand_admin_api_brand.yml"
+    - "@LoevgaardSyliusBrandPlugin/Resources/config/grids/loevgaard_sylius_brand_admin_api_brand_image.yml"
+
 sylius_resource:
```

Comment:
Should this be done with Sylius' AbstractExtension thing we use everywhere else?

Reply:
As far as we already had this config that should be included in app - I've just reworked grids and used include. 

Injecting that config in extension is good approach (as far as it not force app developer to do extra step to include app config). But from other side - readability is worse and too big chance to make typo in big configs.

Reply:
Oh, sorry, I was thinking in my previous comment - you meant this approach: https://github.com/Sylius/CustomerOrderCancellationPlugin/blob/master/src/DependencyInjection/SyliusCustomerOrderCancellationExtension.php#L32

Thing what you meant and we using everywhere (`AbstractExtension`) is designed to work with entities defined at `/Model` directory and I've not tried to make it work with entities at `/Entity` directory (it not work by default, AFAIK). So I've made that changes at another WIP PR for future 2.0 release: https://github.com/loevgaard/SyliusBrandPlugin/pull/35/commits/397e054dfe43926bba34190013a970414a32857a

Reply:
Okay, that's cool